std-bits/base/src/main/java/org/enso/base/Array_Utils.java

```diff
@@ -15,8 +21,37 @@ public static byte[] ensureByteArray(byte[] input) {
     return input;
   }
 
+  /** Creates new builder */
+  public static <V> Array_Utils<V> newBuilder(int capacity) {
```

Comment:
Can we call this class something along the lines `Array_Builder`? `Array_Utils` sounds like a more generic tool whereas this class is exactly that - a builder.

Reply:
Done in [6752572](https://github.com/enso-org/enso/pull/3667/commits/675257210ce726d69bcb50995ce94537dbebe8d0)